CMake/utils/kwiver-utils-modules.cmake

```diff
@@ -23,7 +23,7 @@ define_property(GLOBAL PROPERTY kwiver_modules_enabled
 # enabled.
 #
 # The intended purpose for modules is ... (what is a module? How does
-# it differ from a library?)
+# it differ from a library?) Would this be like adding arrows? or arrows/core?
```

Comment:
hmm... these comments are odd.  It sounds like the purpose of "modules" is unknown.  Is the kwiver_add_module function still used or is this a relic of the way things used to work in MAP-Tk?